docs/devctr-image.md

```diff
@@ -21,6 +21,8 @@ registry. The Firecracker CI suite must also be updated to use the new image.
 
 ### `x86_64`
 
+1. Make sure you are using AWS CLI >=2 by running `aws --version`.
```

Comment:
Is this really required? I mostly run awscli v1 and don't recall problems.

Reply:
I remember a related issue but it does not spring to mind at the moment. I have removed this line.